src/useStore.ts

```diff
@@ -239,6 +239,8 @@ export async function executeCode({
       errors = [e]
       logs = []
       if (e.msg === 'file is empty') engineCommandManager.endSession()
+      if (e.msg.slice(0, 27) === 'found list of unkown tokens')
```

Comment:
We never want to compare error strings like this. Inspecting strings is not a good way to identify errors -- the problem is that now if you change the error message, you'll need to update this too.

Instead, I would change this to check if the error is a lexical error.